modules/tinymce/src/themes/silver/main/ts/ui/throbber/Throbber.ts

```diff
@@ -25,27 +27,54 @@ const renderSpinner = (providerBackstage: UiFactoryBackstageProviders): AlloySpe
       dom: DomFactory.fromHtml('<div class="tox-spinner"><div></div><div></div><div></div></div>')
     }
   ],
-  behaviours: Behaviour.derive([
-    // Trap the "Tab" key and don't let it escape.
-    Keying.config({
-      mode: 'special',
-      onTab: () => Optional.some(true),
-      onShiftTab: () => Optional.some(true)
-    }),
-    Focusing.config({ })
-  ])
 });
 
-const toggleThrobber = (comp: AlloyComponent, state: boolean, providerBackstage: UiFactoryBackstageProviders) => {
+const focusBusyComponent = (throbber: AlloyComponent): void =>
+  Composing.getCurrent(throbber).each((comp) => Focus.focus(comp.element));
+
+// When the throbber is enabled, prevent the iframe from being part of the sequential keyboard navigation when Tabbing
+// TODO: TINY-7500 Only works for iframe mode at this stage
+const toggleEditorTabIndex = (editor: Editor, state: boolean) => {
+  Optional.from(editor.iframeElement)
+    .map(SugarElement.fromDom)
+    .each((iframe) => {
+      if (state) {
+        Attribute.set(iframe, 'tabindex', -1);
+      } else {
+        Obj.get(Settings.getIframeAttrs(editor), 'tabindex')
```

Comment:
I'm not sure this is the best way to go about this... while it will restore the original value, what if someone had changed it after initialization? It might be better to store the original as a data attribute and restore using that instead.